share/completions/dpkg-reconfigure.fish

```diff
@@ -8,7 +8,7 @@ complete -x -f -c dpkg-reconfigure -s h -l help     --description 'Display help'
 # General options
 complete -f -c dpkg-reconfigure -s f -l frontend -r -a "dialog readline noninteractive gnome kde editor web" --description 'Set configuration frontend'
 complete -f -c dpkg-reconfigure -s p -l priority -r -a "low medium high critical" --description 'Set priority threshold'
-complete -f -c dpkg-reconfigure -l default-priority --description 'Use default priority threshold'
+complete -f -c dpkg-reconfigure -l default-priority --description "Use current default ("(echo get debconf/priority | debconf-communicate | string match -r '\w*$')") priority threshold"
```

Comment:
>I like the indexing a bit more, but I'd go with that regex if you prefer.

The problem with the indexing is also that we print an ugly error for an index > 1 (or < -1) if it doesn't exist - `echo $a[2]`. So the "[2]" is unworkable, currently.

For that reason using indexes on command substitutions is not something I get in the habit of doing - I wouldn't call it "idiomatic fish". This is a minor style nit, though.